configs/windows/Agent/nanoserver/NanoServer2022.Dockerfile

```diff
@@ -0,0 +1,102 @@
+# The list of required arguments
+# ARG nanoserverImage
+# ARG powershellImage
+# ARG teamcityWindowsservercoreImage
+
+# Id teamcity-agent
+# Tag ${versionTag}-${tag}
+# Tag ${latestTag}
+# Tag ${versionTag}
+# Platform ${windowsPlatform}
+# Repo ${repo}
+# Weight 2
+
+## ${agentCommentHeader}
+
+# @AddToolToDoc [${jdkWindowsComponentName}](${jdkWindowsComponent})
+# @AddToolToDoc ${powerShellComponentName}
+# @AddToolToDoc [${gitWindowsComponentName}](${gitWindowsComponent})
+# @AddToolToDoc [${dotnetWindowsComponentName}](${dotnetWindowsComponent})
+
+# Based on ${powershellImage} 3
+FROM ${powershellImage} AS dotnet
+
+# On some agents, Windows 2022 requires administrator permissions to modify "C:/" folder within ...
+# ... PowerShell container.
+USER ContainerAdministrator
+
+COPY scripts/*.cs /scripts/
+SHELL ["pwsssssssh", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
pwsh?

Reply:
This is a great catch, thank you! 
Please, excuse me - the line was accidentally added when from the upstream pull request, specifically: the implementation of .NET-based generation of Dockerfiles for the new images.
Fixed it.